patterns/atoms/buttons/primary.php

```diff
@@ -1,2 +1,17 @@
-<p><a href="#" class="a__btn a__btn__primary">Click Me!</a></p>
-<p><a href="#" class="a__btn a__btn__primary" disabled="disabled">Disabled</a></p>
+<?php
```

Comment:
A coupple of co

- I think in this case is better just to have a single file called `button` and have an extra param to control the type of button that is going to be used based on the `class_name` parameter ideally this parameter should be an array so we can pass any number of classes to the button that we want to use.

- Also I don't think we require the `<p></p>` tag around the button.

- One last thing I think there are situations when you require a `<button>` so I guess we can call this `link-as-button` or something like that and have another atom that uses the `<button>` HTML tag instead one that does not require to have an  `url` param.